server/server.js

```diff
@@ -1,15 +1,158 @@
 const express = require("express");
 const app = express();
 const port = process.env.PORT || 5000;
+const cors = require("cors");
+
+const dotenv = require("dotenv");
+dotenv.config();
+const { Pool } = require("pg");
+
+const db = new Pool({
+  connectionString: process.env.DB_URL,
+  ssl: { rejectUnauthorized: false },
+});
+const bodyParser = require("body-parser");
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
I think the second call here cancels out the first one - there's only one kind of body parser you're registering, and it will use the last one you registered - you probably want to remove the first line here.

Reply:
Thank you. I deleted the first line)